static/app/views/performance/browser/resources/index.tsx

```diff
@@ -143,9 +151,10 @@ function PageSelector({value}: {value?: string}) {
   const location = useLocation();
   const {data: pages} = useResourcePagesQuery();
 
-  const options: Option[] = pages.map(page => {
-    return {value: page, label: page};
-  });
+  const options: Option[] = [
+    {value: '', label: 'All'},
+    ...pages.map(page => ({value: page, label: page})),
```

Comment:
Would be super cool to get the favicon of the site if possible (should be available under `DOMAIN_NAME/favicon.ico` - see https://avatars.githubusercontent.com/favicon.ico).

Might be jank in darkmode though 🤔

Reply:
Wow, that's an awesome idea!

Reply:
I'm gonna go through this and all these other PR idea and write them all down in one task!